libs/go-libs/service/app.go

```diff
@@ -13,22 +15,20 @@ import (
 
 const DebugFlag = "debug"
 
-func BindFlags(cmd *cobra.Command) {
+func InitCliFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().Bool(DebugFlag, false, "Debug mode")
 	cmd.PersistentFlags().Bool(logging.JsonFormattingLoggerFlag, false, "Format logs as json")
-
-	if err := viper.BindPFlags(cmd.PersistentFlags()); err != nil {
-		panic(err)
-	}
+	licence.InitCLIFlags(cmd)
```

Comment:
This will be also added to membership

Reply:
@gfyrag Is it okay to add the licence system in this library ? It will be added to all service, but it's the cleanest way to stop the app gracefully when we have the licence failed

Reply:
I think the tooling to allow any service to exists with any exit code is ok.
But I think the init of the licence module should not be in this package. I think each app should add the call to the root.go. It will remove the flags from non EE services.

@Dav-14 membership is not OSS, so not the subject actually, it will maybe handle differently.